### PR TITLE
Add text editor shortcuts

### PR DESCRIPTION
**Pull request by 8kdesign**

### CODE REVIEW

File: src/main/java/seedu/duke/editor/ShortcutListener.java [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    /**
+     * Checks if keys for save shortcut are pressed.
+     * Calls save file method if true.
+     */
+    void checkSaveShortcut() {
+        if (isSPressed && isCtrlPressed) {
+            isCtrlPressed = false;
+            isSPressed = false;
+            //Save
+
```
Comment (H-horizon) [inline src/main/java/seedu/duke/editor/ShortcutListener.java:68]: Method name: saveTextToFile() ;

File: src/main/java/seedu/duke/editor/ShortcutListener.java [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+    }
+
+    @Override
+    public void keyReleased(KeyEvent e) {
+        switch (e.getKeyCode()) {
+        case KeyEvent.VK_CONTROL: {
+            isCtrlPressed = false;
+            break;
+        }
+        case 83: {
```
Comment (H-horizon) [inline src/main/java/seedu/duke/editor/ShortcutListener.java:51]: Refactor magic number

**Review by H-horizon — APPROVED** [review]

File: src/main/java/seedu/duke/editor/ShortcutListener.java (hunk 2 above)
Comment (8kdesign) [reply src/main/java/seedu/duke/editor/ShortcutListener.java:51]: Oops forgot about this

File: src/main/java/seedu/duke/editor/ShortcutListener.java (hunk 1 above)
Comment (8kdesign) [reply src/main/java/seedu/duke/editor/ShortcutListener.java:68]: Can't add since method not in code yet,

**Pull request merged**